chore: update readme [skip ci] (#121)
posva/vite-tailwind-starter: README.md

# Vite + Tailwind CSS Starter

> A [Tailwind CSS](http://tailwindcss.com/) starter template for [Vite](https://github.com/vitejs/vite)

## Tailwind UI Users

> If you have access to [Tailwind UI](https://tailwindui.com), you can follow the following steps to add it:

Reviewer: ```suggestion
If you have access to [Tailwind UI](https://tailwindui.com), follow these steps to add it:
```

1. Install first-party plugins:

```sh
yarn add @tailwindcss/forms @tailwindcss/typography @tailwindcss/aspect-ratio
```

> 2. Add the plugins in `tailwind.config.js` without changing anything else:

Reviewer: ```suggestion
2. Add the plugins to `tailwind.config.js`:
```

# Vite + Tailwind CSS starter
# Vite + Tailwind CSS Starter

> Note if you have access to [Tailwind UI](https://tailwindui.com), you can follow the following steps to add it:

Reviewer: Can you add the part from https://tailwindui.com/changes-for-v2 _Add first-party plugins-?

> A [Tailwind CSS](http://tailwindcss.com/) starter template for [Vite](https://github.com/vitejs/vite)

Reviewer: ```suggestion
> A [Tailwind CSS](http://tailwindcss.com/) starter template for [Vite](https://github.com/vitejs/vite)
```
